### PR TITLE
stop doing local imports

### PR DESCRIPTION
**Pull request by nickpell**
Ran ```futurize . -w -f libfuturize.fixes.fix_absolute_import```

Seems like a good prelude to running ```python-modernize -w -f libmodernize.fixes.fix_import``` since the first command seems slightly riskier than the second (which adds ```from __future__ import absolute_import``` to all files).

@dimagi/py3

### CODE REVIEW

File: corehq/ex-submodules/auditcare/decorators/__init__.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
-from login import watch_login
-from login import watch_logout
+from __future__ import absolute_import
+from .login import watch_login
```
[the remark this answers is not in the document]
Comment (nickpell) [reply corehq/ex-submodules/auditcare/decorators/__init__.py]: ignoring

File: corehq/ex-submodules/auditcare/decorators/__init__.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
...
+from .login import watch_logout
```
[the remark this answers is not in the document]
Comment (nickpell) [reply corehq/ex-submodules/auditcare/decorators/__init__.py]: ignoring

File: custom/apps/gsid/reports/__init__.py [hunk 3]
```diff
@@ -11,7 +12,7 @@
     GSIDSQLTestLotsReport,
     PatientMapReport,
 )
-from util import get_unique_combinations
+from .util import get_unique_combinations
```
[the remark this answers is not in the document]
Comment (nickpell) [reply custom/apps/gsid/reports/__init__.py]: ignoring

File: corehq/ex-submodules/auditcare/decorators/__init__.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (nickpell) [reply corehq/ex-submodules/auditcare/decorators/__init__.py]: ignoring

File: scripts/rebuildstaging.py [hunk 4]
```diff
@@ -36,8 +37,8 @@
 import gevent
 
 from fabric.colors import red
-from sh_verbose import ShVerbose
-from gitutils import (
+from .sh_verbose import ShVerbose
+from .gitutils import (
```
[the remark this answers is not in the document]
Comment (nickpell) [reply scripts/rebuildstaging.py]: ignoring

File: scripts/rebuildstaging.py [hunk 5]
```diff
@@ -36,8 +37,8 @@
 import gevent
 
 from fabric.colors import red
-from sh_verbose import ShVerbose
-from gitutils import (
+from .sh_verbose import ShVerbose
```
[the remark this answers is not in the document]
Comment (nickpell) [reply scripts/rebuildstaging.py]: ignoring

File: scripts/couchdb/replicate_db_to_new_node.py [hunk 6]
```diff
@@ -1,6 +1,7 @@
+from __future__ import absolute_import
 import sys
 
-from utils import (
+from .utils import (
```
[the remark this answers is not in the document]
Comment (nickpell) [reply scripts/couchdb/replicate_db_to_new_node.py]: ignoring

**Comment by nickpell** [comment]
tests passed. ignoring the stickler comments above.

**Review by millerdev — APPROVED** [review]

**Pull request merged**